addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -866,7 +866,7 @@
         data-selector=".note-editable > div:not(.o_layout), .note-editable .oe_structure > div, td, th"
         data-exclude=".o_mail_no_resize, .o_mail_no_options"/>
 
-    <div data-selector=".note-editable > div:not(.o_layout), .note-editable .oe_structure > div, td, th"
+    <div data-selector="table, td, th"
```

Comment:
> From my test desktop outlook (or mail app on recent windows) want background color on table/tr/td/th and not on div, so I changed the selector.

Not entirely stable change, especially if editing previously made templates right? Maybe another solution would be to use `background: inherit` on tables (maybe only for your problematic mail client)... if the mail client understands inherit ? :grimacing:

Reply:
I have tested (putting background:inherit on all contained table, tr, td) and it doesn't seem to give background-color of div to contained table elements.

A solution for the issue might  be to add:

```diff
- <div data-selector=".note-editable > div:not(.o_layout), .note-editable .oe_structure > div, td, th"
+ <div data-selector=".note-editable > div:not(.o_layout)[style*="background-color], .note-editable .oe_structure > div[style*="background-color], table, td, th"
```

so the option is still shown for <div/> that already have an inline `background-color`